policy/controller.go

```diff
@@ -54,11 +54,22 @@ func (p runParameters) SetOutputs(op io.Writer, gh GHoutput) error {
 		gh.TestVariations[p["job"]+"_conf"] = []string{"sha256", "murmur128"}
 		gh.TestVariations[p["job"]+"_db"] = []string{"mongo7", "postgres15"}
 		gh.TestVariations[p["job"]+"_cache_db"] = []string{"redis7"}
-		gh.TestVariations["pump"] = []string{"$ECR/tyk-pump:master"}
-		gh.TestVariations["sink"] = []string{"$ECR/tyk-sink:master"}
-		gh.Exclusions = []map[string]string{
-			{"db": "mongo7", "conf": "murmur128"},
-			{"db": "postgres15", "conf": "sha256"},
+		if p["repo"] == "tyk-ci" {
```

Comment:
This if should not be here. It should be driven by the config file.

Reply:
@alephnull I suggest we pair on this , I will setup and schedule